Sources/Fuzzilli/FuzzIL/JsOperations.swift

```diff
@@ -2464,6 +2464,17 @@ class BindMethod: JsOperation {
     }
 }
 
+final class LoopNestedContinue: JsOperation {
```

Comment:
Maybe here would be a good place for a short comment describing how this operation works, in particular how `depth` is interpreted (e.g. that it wraps around if there are fewer open loops)